src/iotjs_util.c

```diff
@@ -43,28 +43,29 @@ iotjs_string_t iotjs_file_read(const char* path) {
     ptr = buffer + read;
   }
 #else
-  size_t read = fread(buffer, 1, len, file);
+  size_t read = fread(buffer, 1, (size_t)len, file);
 #endif
   IOTJS_ASSERT(read == (size_t)len);
 
   *(buffer + len) = 0;
 
   fclose(file);
 
-  iotjs_string_t contents = iotjs_string_create_with_buffer(buffer, len);
+  iotjs_string_t contents =
+      iotjs_string_create_with_buffer(buffer, (uint32_t)len);
```

Comment:
I think `len` should have `size_type`. Basically every time using some variable if we have to cast with some specific type, then I think this variable should have declared with the type to cast at the first time.